docker_daemon/check.py

```diff
@@ -179,10 +179,15 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             raise Exception("Docker check only supports one configured instance.")
         AgentCheck.__init__(self, name, init_config,
                             agentConfig, instances=instances)
-
         self.init_success = False
         self._service_discovery = agentConfig.get('service_discovery') and \
             agentConfig.get('service_discovery_backend') == 'docker'
+
+        self.collect_labels_as_tags = agentConfig.get('docker_labels_as_tags', ','.join(DEFAULT_LABELS_AS_TAGS))
```

Comment:
just remove `, ','.join(DEFAULT_LABELS_AS_TAGS)` and let `get` return a nil. In the following `else` just do `self.collect_labels_as_tags = DEFAULT_LABELS_AS_TAGS`